vaadin-icons-flow-parent/vaadin-icons-flow/src/main/java/com/vaadin/flow/component/icon/SvgIcon.java

```diff
@@ -43,6 +43,19 @@ public SvgIcon(String src) {
         setSrc(src);
     }
 
+    /**
+     * Creates an SVG icon with the given source and symbol
+     *
+     * @param src
+     *            the SVG file path
+     * @see #setSrc(String)
+     * @see #setSymbol(String)
+     */
+    public SvgIcon(String src, String symbol) {
```

Comment:
Should there also be constructors and setters for `src` only, so you wouldn't always need to pass `null` as the second parameter

Reply:
Ah, nevermind, they're already there :)